### PR TITLE
docs: add project to `project_created` event, fix schema

### PR DESCRIPTION
**Pull request by malta895**
## Pull Request Type

- [x] Documentation content changes
- [ ] Bugfix / Missing Redirects
- [ ] Docusaurus site code changes

## PR Checklist

- [x] The commit message follows our guidelines included in the [CONTRIBUTING.md](../blob/main/CONTRIBUTING.md#how-to-submit-a-pr)
- [x] All tests of Lint, cspell and check-content pass. ([How to launch them?](../blob/main/CONTRIBUTING.md#content-checks))
- [x] No sensitive content has been committed

### CODE REVIEW

**Review by mattia-fumo — APPROVED** [review]
lgtm

File: docs/console/company-configuration/events.mdx [hunk 1]
```diff
@@ -41,7 +47,9 @@ This event will be triggered when a Project is created successfully.
   "eventTimestamp": 123456,
   "payload": {
     "tenantId": "my-company-id",
-    "projectId": "0000000000000001"
+    "projectId": "0000000000000001",
+    "sourceTemplateId": "0000000000000002",
+    "project": {...}
```
Comment (Danielecina) [inline docs/console/company-configuration/events.mdx:52]:
```suggestion
    // See the Project resource tab
    "project": {...}
```

File: docs/console/company-configuration/events.mdx [hunk 2]
```diff
@@ -542,235 +1025,4 @@ This event does not guarantee that the user in question already had access to th
 ```
 
 </TabItem>
-</Tabs>
-
-
-## Entities
-
-### Project
```
Comment (Danielecina) [inline docs/console/company-configuration/events.mdx:550]: Dov'è stata messa lei?

Comment (malta895) [reply docs/console/company-configuration/events.mdx:550]: C'è, i due eventi tornano entità diverse di project e quindi ne ho messe due separate in ciascun evento

**Review by jacoposte — APPROVED** [review]

**Pull request merged**